src/components/utils/registerCustomKeysets.test.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+
+import type {KeysetData} from '@gravity-ui/i18n';
+
+import {render, screen} from '../../../test-utils/utils';
+import {i18n} from '../../i18n';
+import {Dialog} from '../Dialog';
+import {Pagination} from '../Pagination';
+
+import {Lang, configure} from './configure';
+import {registerCustomKeysets} from './registerCustomKeysets';
+
```

Comment:
Is it ok, that I don't have `describe`, or should I encapsulate everything in it? I haven't done it, because jest splits tests by files anyway